src/main/java/org/usfirst/frc4048/common/apriltags/ApriltagInputs.java

```diff
@@ -0,0 +1,33 @@
+package org.usfirst.frc4048.common.apriltags;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class ApriltagInputs implements LoggableInputs {
+    public double[] timestamp = new double[0];
```

Comment:
The reason I did not use a record for this class is because other people could use it as an example for creating `LoggableInputs` for subsystems. Students with less experience in recent versions of java most likely don't know what a record is. I tried only to use records for backend related stuff that hopefully would not have to be changed or used as a template.